walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,24 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
+    Greeter greeter = new Greeter();
+
+    String greeting = greeter.greet("   Ada   ");
+
+    // Whitespace should be trimmed
```

Comment:
nit - i think you can drop these comments, or update your test method names to make it clear what the test is asserting. typically we look more at the test name to understand what the test is asserting. this has the advantage of not having to update two places if the test logic changes.

Reply:
Ok, I'll delete the comments, and possibly adjust the names if they are inadequate.